src/components/App/AppError/AppError.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC, memo } from 'react';
+import { Loader } from '../../Loader';
+import { ErrorType } from '../../../types/ErrorType';
+
+interface Props {
+  error: ErrorType;
+}
+
+export const AppError: FC<Props> = memo((props) => {
+  const { error } = props;
+
+  return (
+    <>
```

Comment:
mb would it be better if we put a wrapper here?
Bc you can break components where you will use this error
```suggestion
    <div>
```

Reply:
Up to you